frontend/src/features/map_planning/components/Map.tsx

```diff
@@ -146,15 +143,14 @@ export const Map = ({ layers }: MapProps) => {
         <BaseLayer
           opacity={untrackedState.layers.base.opacity}
           visible={untrackedState.layers.base.visible}
-          nextcloudImagePath={trackedState.layers.base.nextcloudImagePath}
-          pixelsPerMeter={trackedState.layers.base.scale}
-          rotation={trackedState.layers.base.rotation}
+          listening={selectedLayer.type_ === LayerType.Base}
         />
         <PlantsLayer
           visible={untrackedState.layers.plants.visible}
           opacity={untrackedState.layers.plants.opacity}
           listening={selectedLayer.type_ === LayerType.Plants}
         ></PlantsLayer>
+        <BaseMeasurementLayer />
```

Comment:
I think this needs to go to the top of the list to catch the click events.

Reply:
Layers on the bottom will be shown drawn on top.
Besides, I did not notice any issues regarding click events.
Did I miss anything?

Reply:
@badnames did you try to change the order?